go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/opencontainers/image-spec v1.0.2
 	github.com/openshift/api v0.0.0-20220525145417-ee5b62754c68
 	github.com/openshift/cluster-api-provider-agent/api v0.0.0-20220227135922-dd6353f609dc
+	github.com/openshift/cluster-node-tuning-operator v0.0.0-20220614214129-2c76314fb3cc
```

Comment:
it'd be nicer to review splitting vendoring into it's own commit.

Reply:
I split the vendoring into its own commit, thanks.